### DISCUSSION TITLE
[Feature Request] Dark Theme

### DISCUSSION THREAD

**Discussion by Dvalin21** [tobychui/zoraxy#81] [Ideas]
Just started using this awesome application today. Thank you for all the work that you and however else put into this. Requesting for change of theme function. More specifically Dark/Black theme. Make it a little easier on the eyes. Thanks in advance.

**Comment by tobychui** [comment]
I think this should be moved to issue instead of discussion. Would you mind closing this and opening a new issue as a formal feature request?

**Reply by Dvalin21** [reply]
Yeah no problem!!
